### PR TITLE
doc: add docs and default config and sample deployment manifest

### PR DESCRIPTION
**Pull request by kezhenxu94**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,22 @@
-# Kubernetes Event Exporter
+# SkyWalking Kubernetes Event Exporter
```
Comment (wu-sheng) [inline README.md:1]: Let's add logo, twitter, star badges here. Or CI?

File: assets/default-config.yaml [hunk 2]
```diff
@@ -17,13 +17,20 @@
 #
 
 filters:
-  - namespace: istio-system
-    exporters:
+  - reason: ""     # filter events of the specified reason, regular expression like "Killing|Killed" is supported.
```
Comment (wu-sheng) [inline assets/default-config.yaml:20]: Should we set some recommended reasons and other configurations? Such as for mesh case?

Comment (kezhenxu94) [reply assets/default-config.yaml:20]: > Should we set some recommended reasons and other configurations? Such as for mesh case?

I haven't come up with some recommended reasons and other configurations for now, maybe I can fine-tune these when applying them into real use case later

File: README.md (hunk 1 above)
Comment (kezhenxu94) [reply README.md:1]: Done

File: assets/default-config.yaml (hunk 2 above)
Comment (wu-sheng) [reply assets/default-config.yaml:20]: OK. I read you are adding documentation, I thought you are going to release this.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**